### PR TITLE
Indirect - Remove Error Bars

### LINKED ISSUE #19557: Indirect - Remove error bars
Remove error bars on graphs when 'plot' button on interface is clicked

### PR DESCRIPTION
**Pull request by louisemccann**
Removed error bars displayed as standard when `plot` button pressed on Indirect Interfaces. 

**To test:**

To test:
Run through a couple of indirect interfaces and ensure a graph with error bars is displayed when plot is clicked.
For Example: (requires access to ISIS data archive)
Instrument: IRIS
Indirect > Data Reduction > Calibration
Run No: 26173
Indirect > Data Reduction > Energy Transfer
Run No: 26176
Use 26173_calib workspace as calibration

Indirect > Data Analysis > Elwin
Use 26176_red workspace as input
Indirect > Data Analysis > MSDFit
Use 26176_elwin_eq2 workspace as input

Fixes #19557.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v3.10.0/indirect_inelastic.rst [hunk 1]
```diff
@@ -55,6 +55,7 @@ Improvements
 - OSIRIS diffraction now rebins container workspaces to match the sample workspace
 - :ref:`ISISIndirectDiffractionReduction <algm-ISISIndirectDiffractionReduction>` now fully supports VESUVIO data
 - Inelastic pixel ID's in BASIS instrument definition file grouped into continuous physical pixels.
+- Removed error bars as default
```
Comment (gvardany) [inline docs/source/release/v3.10.0/indirect_inelastic.rst]: Please be more specific here

**Review by gvardany — COMMENTED** [review]
In ISIS Calibration, when I plot the result, they seem to be with errors bars still.

**Comment by SpencerHowells** [comment]
That should be Ok.

**Comment by gvardany** [comment]
you can :shipit: then

**Pull request merged**